Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
+                        expect(callbackWasCalled) == true
+                        expect(secondCallbackWasCalled) == true
+                    }
 
-                    baseObject.trigger("another-event")
+                    it("doesn't execute callback for another event") {
+                        baseObject.on(eventName, callback: callback)
+                        baseObject.trigger("another-event")
 
-                    expect(callbackWasCalled) == false
-                }
-
-                it("Callback should not be called for another context object") {
-                    let anotherObject = BaseObject()
+                        expect(callbackWasCalled) == false
+                    }
 
-                    baseObject.on(eventName, callback: callback)
+                    it("doesn't executes callback for another context object") {
+                        let anotherBaseObject = ConcreteBaseObject()
 
-                    anotherObject.trigger(eventName)
+                        baseObject.on(eventName, callback: callback)
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("once") {
-                it("Callback should be called on event trigger") {
-                    baseObject.once(eventName, callback: callback)
-                    baseObject.trigger(eventName)
+                describe("once") {
+                    it("executes callback function") {
+                        baseObject.once(eventName, callback: callback)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == true
-                }
+                        expect(callbackWasCalled) == true
+                    }
 
-                it("Callback should not be called twice") {
-                    baseObject.once(eventName, callback: callback)
+                    it("doesn't execute callback function twice") {
+                        baseObject.once(eventName, callback: callback)
 
-                    baseObject.trigger(eventName)
-                    callbackWasCalled = false
-                    baseObject.trigger(eventName)
+                        baseObject.trigger(eventName)
+                        callbackWasCalled = false
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
-                }
+                        expect(callbackWasCalled) == false
+                    }
 
-                it("Callback should not be called if removed") {
-                    let listenId = baseObject.once(eventName, callback: callback)
-                    baseObject.off(listenId)
-                    baseObject.trigger(eventName)
+                    it("doesn't execute callback function when it is removed") {
```

Comment:
This ``it`` seems that is holding a context logic, do you think that is necessary to split into a ``context`` > ``it`` ?
It could be:
```swift
context("#once") {
    context("When it is removed") {
        it("doesn't execute callback") {}
    }
}
```